src/app/organizations/collection/collection.component.html

```diff
@@ -44,15 +48,15 @@ <h1><a [routerLink]="['/organizations/', organization.id]">{{ organization.name
       </mat-card>
 
       <mat-card class="alert alert-info" *ngIf="collection?.entries?.length == 0">
-        This collection has no associated entries
+        This collection has no associated entries. To add an entry, navigate to the public page of the tool or workflow you want to add and select `add to collection`
```

Comment:
Should ``add to collection`` be cased properly to match the case when displayed on the button? Or is this a recommendation for how cards are styled?